src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -33,6 +33,8 @@ interface VcsRootLocator {
 interface BuildLocator {
     fun fromConfiguration(buildConfigurationId: BuildConfigurationId): BuildLocator
 
+    fun withNumber(buildNumber: String): BuildLocator
```

Comment:
Why do you need this given that we already have method which find build by its number in TeamCityInstance?